apps/dialtone-documentation/docs/_data/site-nav.json

```diff
@@ -70,7 +96,7 @@
           "planned": true
         },
         {
-          "text": "Design assets",
+          "text": "Design Assets",
```

Comment:
Don't think Assets need to be capitalized.

```suggestion
          "text": "Design assets",
```

Reply:
Per the new writing guidance, menu labels should be title-cased

Reply:
We need to change all the titles then, we were not capitalizing previously, doesn't need to be in this PR, but we should create a ticket to not forgot about it

Reply:
We might as well scrub the current doc site to correct this. I'll do it. Also for action items such as buttons.